mydiscord.py

```diff
@@ -141,8 +139,6 @@ async def on_message(message):
                 textFile.writelines(message.content)
                 textFile.close()
 
-                # --- parse the incoming Discord message
-                status_message = utils.parsemessage()
```

Comment:
This is what was causing the problem.  `utils.parsemessage()` was called twice. After the `utils.song_case_correction()` updated the worshipschedule.txt file with the correct case, `utils.parsemessage()` was called again overwriting it with the original message content.